api/src/main/java/com/staxpayments/api/repository/InvoiceRepository.kt

```diff
@@ -0,0 +1,16 @@
+package com.staxpayments.api.repository
+
+import com.staxpayments.api.models.Invoice
+import com.staxpayments.api.requests.CreateInvoiceBody
+
+interface InvoiceRepository {
+
```

Comment:
we dont need the comment, the naming already reflect what they do

Reply:
That was for me, just forgot to clean up. I have cleaned it up